qt/python/mantidqt/widgets/colorbar/colorbar.py

```diff
@@ -25,14 +26,28 @@
 MIN_LOG_VALUE = 1e-4
 
 
+def register_customized_colormaps():
```

Comment:
This is good. It will now be possible to add more colormaps in the future as needed/requested.